__tests__/lib/apiError.test.ts

```diff
@@ -16,7 +16,7 @@ const response = {
   ],
 };
 
-describe('APIv1Error', () => {
+describe('#APIv1Error', () => {
```

Comment:
unrelated lint warning 🫠

Reply:
what was the warning?

Reply:
![CleanShot 2025-04-28 at 17 37 36@2x](https://github.com/user-attachments/assets/9c93731b-7043-42db-b44d-cda917a0bc6d)

Reply:
not opposed to disabling this rule

Reply:
ah weird, it wants you to do `describe(APIv1Error, () => ...` and it could do `.constructor.name`.

that's... weird. i'm on board with disabling that if you want to PR it to our core standards

Reply:
annoyingly, if i pass the class itself, i get this lint warning:

![CleanShot 2025-04-28 at 17 41 00@2x](https://github.com/user-attachments/assets/ba4a8b76-d59c-4594-94af-b12e4546b8a2)

Reply:
yeah lets just disable `prefer-describe-function-title`. using a function as a test name is **funky**